CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [Unreleased]
+
+## Added
+
+- Added sas 9.4 remote connection support via ITC and the IOM Bridge protocol. This is currently supported for Windows only ([#592](https://github.com/sassoftware/vscode-sas-extension/pull/592))
```

Comment:
nit: Is it clear enough to just say Windows only here? Will any users treat it as 9.4 remote on Windows?
It's actually "VS Code Extensions Host on Windows only". So technically a Mac user can tunnel to a Windows VS Code workspace and run the extension there. I guess we don't want to say too much here in changelog, perhaps say something in connect-and-run.md?

Reply:
I don't think I follow. Currently, this profile type is only supported in a Window environment. If a user is somehow tunneling through (not exactly what that means), they would then be on a Windows environment, right (technically speaking)?

Reply:
I'm thinking if I'm a user who have no knowledge of ITC and IOM, when I see just "Windows only", there're 3 possbilities to me.

1. Does it mean my SAS 9.4 needs to be run on Windows only?
2. Does it mean I have to use VS Code on Windows only?
3. Does it mean both my VS Code and my SAS 9.4 need to be on Windows only?

The anwser of the 3 questions are all "no". I'm thinking it would be great to explain more details

Reply:
By tunneling I'm in general referring to all [remote approaches](https://code.visualstudio.com/docs/remote/remote-overview) provided by VS Code. For Windows, only tunnel and SSH is practical.

Reply:
This conversation leads me to further believe that we need to have a wiki entry with some more intricate details around mapping a users sas "environment" to the underlying technologies that can be used in the connection profiles to carry out the session creation. Not sure if we'd want to do that as part of this effort. @jefurbee  any thoughts here?

Reply:
For now, I've removed the "Windows only" part of this message. I think there's probably a bigger conversation that needs to happen like @smorrisj said.